app/assets/stylesheets/application.css.scss

```diff
@@ -1,23 +1,38 @@
 /*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
+ * This is a manifest file that'll be compiled into application.css, which will
+ * include all the files listed below.
  *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
- * or vendor/assets/stylesheets of plugins, if any, can be referenced here using a relative path.
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets,
+ * vendor/assets/stylesheets, or vendor/assets/stylesheets of plugins, if any,
+ * can be referenced here using a relative path.
  *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any styles
- * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
- * file per style scope.
+ * You're free to add application-wide styles to this file and they'll appear at
+ * the bottom of the compiled file so the styles you add here take precedence
+ * over styles defined in any styles defined in the other CSS/SCSS files in this
+ * directory. It is generally better to create a new file per style scope.
  *
  *= require_tree .
  *= require_self
  */
 
+@import 'bootstrap';
+
 #header {
   margin-top: 10px;
 }
 
-#middle p {
-  font-size: 22px;
+.field_with_errors {
+  @extend .has-error;
+}
+
+.card_image p {
+  margin-bottom: 0px;
+}
+
+.card_remove_image {
+  margin-left: 20px;
+}
+
+.block_user_current_block_id {
```

Comment:
Selector `block_user_current_block_id` should be written in lowercase with hyphens